stable/mysql/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: mysql
-version: 0.9.3
+version: 0.9.4
```

Comment:
Please bump minor version since this introduce new features

Reply:
Bumped to 0.10.0.